awx/settings/defaults.py

```diff
@@ -1003,3 +1003,9 @@
 
 # This is overridden downstream via /etc/tower/conf.d/cluster_host_id.py
 CLUSTER_HOST_ID = socket.gethostname()
+
+# License compliance for total host count. Possible values:
+# - '': No model - Subscription not counted from Host Metrics
+# - 'unique_managed_hosts': Compliant = automated - deleted hosts (using /api/v2/host_metrics/)
+# - 'unique_managed_hosts_monthly': TBD: AoC on Azure (now equal to '')
+SUBSCRIPTION_USAGE_MODEL = ''
```

Comment:
Will "unique_managed_hosts_monthly" be implemented with this change?  I've been informed to set SUBSCRIPTION_USAGE_MODEL = 'unique_managed_hosts_monthly' by @Ladas but looking at this PR not sure that will be the way to go for 2.4.  I tried adding it in our current 2.3 version to be ready in advance, but controller was not happy (thought maybe it would be ignored).

Reply:
The "unique_managed_hosts_monthly" will not show anything for 2.4. (most likely, we might be able to backport something to 2.4.x, if we'll get the DB changes in 2.4)